README.md

```diff
@@ -7,80 +7,62 @@
 [![Docs](https://img.shields.io/badge/docs-rust-brightgreen?style=for-the-badge)](https://project-oak.github.io/oak)
 
-The goal of Project Oak is to create a specification and a reference
-implementation for the secure transfer, storage and processing of data.
+The goal of Project Oak is to provide infrastructure to transfer, store and
+process data in a secure and transparent way.
 
-In present computing platforms (including virtualized, and cloud platforms),
-data may be encrypted at rest and in transit, but they are exposed to any part
-of the system that needs to process them. Even if the application is securely
-designed and data are encrypted, the operating system kernel (and any component
-with privileged access to the machine that handles the data) has unrestricted
-access to the machine hardware resources, and can leverage that to bypass any
-security mechanism on the machine itself and extract secret keys and data.
-
-[Oak Functions](/oak_functions) is a server binary that exposes an API (over
-gRPC / HTTP) that allows clients to send data, and the server to execute
-untrusted business logic code over client-provided data in a controlled way,
-enforcing strong privacy guarantees about its execution.
-
-In order for these guarantees to be transferred to remote clients, Oak Functions
-is expected to be run in a
-[Trusted Execution Environment (TEE)](https://en.wikipedia.org/wiki/Trusted_execution_environment),
-so that clients can perform [Remote Attestation](./docs/remote-attestation.md)
-in order to establish that a legitimate version of the Oak trusted runtime is
-indeed running server side, before actually sending any data to it.
-
-Further information is included in the following documents:
-
-- [Oak Development](docs/development.md) covers practical steps for getting a
-  development Oak system up and running.
-- [Oak Functions](oak_functions/README.md) presents our computing platform for
-  developing stateless applications in a privacy preserving way.
+To do so, Oak relies on running _Oak Trusted Enclave Applications_ in a
+[Trusted Execution Environment (TEE)](https://en.wikipedia.org/wiki/Trusted_execution_environment).
+An example of a Trusted Enclave Application is
+[Oak Functions](/oak_functions/README.md). The Trusted Enclave Application can
+provide the client cryptographically attested evidence of the security
+properties by [Remote Attestation](./docs/remote-attestation.md). Together with
+[Transparent Release](https://github.com/project-oak/transparent-release) this
+binds the open-source source code to the remotely attested binary running inside
+the TEE. In order to feasibly review all the source code running inside the TEE,
+and minimize our trusted computing base, Oak provides the following
+infrastructure: [stage 0](/stage0/),
+[Oak Restricted Kernel](/oak_restricted_kernel/) and a controlled communications
+interfaces---the [Oak Comms Channel](/oak_channel/) and [microRPC](/micro_rpc/).
```

Comment:
I don't think the `---` is rendered correctly; anyways, probably a bullet list here may be better (typographically)

Reply:
Replaced by i.e.,